src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Ball {
+    private int number;
+    private String color;
```

Comment:
Consider using an enum for the color instead of a String to represent a predefined set of colors. This can help prevent errors related to invalid color strings and make the code more maintainable.